data/Himanshuch8055.json

```
@@ -0,0 +1,53 @@
+{
+  "name": "Himanshu Chauhan",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open source Advocate | Programmer | Content Writer | Developer | Competitive Coder",
+  "tags": ["Open Source", "Content Writer", "Blogging", "Web3", "Developers", "Competitive Coding"],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/himanshuch8055" },
+    { "icon": "FaGithub", "url": "https://github.com/Himanshuch8055" },
+    { "icon": "FaInstagram", "url": "https://www.instagram.com/himanshuchauhan8055/" },
+    { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/himanshu-chauhan-693404226/" },
+    { "icon": "FaFacebook", "url": "https://www.facebook.com/himanshuchauhan8055" },
+    { "icon": "FaYoutube", "url": "https://www.youtube.com/channel/UCoBSwNEZRZ1ROny1RNACh7A" },
+    { "icon": "FaDiscord", "url": "https://discord.gg/https://discord.gg/deYQcTGk" },
```

Wrong link here.
```suggestion
    { "icon": "FaDiscord", "url": "https://discord.gg/deYQcTGk" },
```

---

> Wrong link here.
> ```suggestion
>     { "icon": "FaDiscord", "url": "https://discord.gg/deYQcTGk" },
> ```
> 

Must've missed that! Thanks 🙌